src/main.rs

```diff
@@ -315,6 +362,76 @@ fn open_config(path: &Path) -> Option<File> {
     }
 }
 
+fn validate(opts: &Validate, root_opts: &RootOpts) -> exitcode::ExitCode {
+    if root_opts
+        .config_path
+        .to_str()
+        .map_or(false, |s| s != "/etc/vector/vector.toml")
```

Comment:
So it has a default impl above for RootOps so you could do

```rust
let opts = RootOpts::default()
let path = opts.config_path
```

What do you think?

Reply:
Yessss that was it. I couldn't seem to get default to work but `RootOpts::from_iter(vec![""])` did the trick.